tools/odbc/prepared.cpp

```diff
@@ -109,7 +96,7 @@ SQLRETURN SQLNumParams(SQLHSTMT StatementHandle, SQLSMALLINT *ParameterCountPtr)
 SQLRETURN SQLBindCol(SQLHSTMT StatementHandle, SQLUSMALLINT ColumnNumber, SQLSMALLINT TargetType,
                      SQLPOINTER TargetValuePtr, SQLLEN BufferLength, SQLLEN *StrLen_or_IndPtr) {
 	return WithStatementPrepared(StatementHandle, [&](OdbcHandleStmt *stmt) {
-		auto col_nr_internal = ColumnNumber - 1;
+		size_t col_nr_internal = ColumnNumber - 1;
```

Comment:
Could you enable the clang-tidy checker on the ODBC driver by adding `-DBUILD_ODBC_DRIVER=TRUE` to the `tidy-check` build in the Makefile? There are a number of places where coding conventions (e.g. variable/function names) are inconsistent and it would be great to keep them consistent everywhere.